### DISCUSSION TITLE
Why does Tumblthree only save one image per post?

### DISCUSSION THREAD

**Discussion by yoiyoiyoi** [TumblThreeApp/TumblThree#481] [Q&A (Question & Answer)]
I'm trying to save all the posts from a tumblr blog. However, if tumblthree comes across a post with multiple images, it saves only the first and completely skips all the rest. The same thing happens if I try to download by tag. I want to save every image in posts, not just the top one. How do I do this? Thank you!

**Comment by thomas694** [comment]
Do you use the latest version?
Did you make any changes of the default settings?
